Graphics/Vulkan/include/NovelRT/Graphics/Vulkan/Utilities/MemoryAccessMode.hpp

```diff
@@ -3,32 +3,37 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#include <vulkan/vulkan.h>
 #include <NovelRT/Graphics/GraphicsMemoryAccessMode.hpp>
-#include <NovelRT/Utilities/Misc.h>
+#include <NovelRT/Utilities/Operators.hpp>
+
```

Comment:
nit: random newline

Reply:
This isn't random - it's separating the includes based on where they're from. Ours, then stdlib, then thirdparty.